core/src/main/java/com/linecorp/armeria/server/DefaultServiceRequestContext.java

```diff
@@ -362,12 +362,12 @@ public void setRequestTimeoutMillis(long requestTimeoutMillis) {
         }
         if (this.requestTimeoutMillis != requestTimeoutMillis) {
             this.requestTimeoutMillis = requestTimeoutMillis;
-            final RequestTimeoutChangeListener listener = requestTimeoutChangeListener;
+            final RequestTimeoutController listener = requestTimeoutController;
```

Comment:
`listener` -> `requestTimeoutController`?

Reply:
Oops... thanks for catching it.